client/encoding_test.go

```diff
@@ -32,3 +33,12 @@ func TestUnmarshalMalformedErrorResponse(t *testing.T) {
 			"got", res.Error)
 	}
 }
+
+func TestUnmarshalSampleMessage(t *testing.T) {
+	msg := `{"Error":100,"DirectoryResponse":{"AP":{"TreeNonce":"dmriJjZCIs451/6e6OWdEhukqf35szGK0MDP5tqStoU=","PrunedTree":[[248,166,186,56,249,219,214,188,80,58,222,92,254,175,220,201,255,212,82,86,184,177,28,122,242,237,252,205,37,56,178,164]],"LookupIndex":"HzdAcxzkVDEdCnrc0Rq6KZ7zewuqgy6BZrBBXkbPiL4=","VrfProof":"ZIbhTZFn0l6K8NISjBpOo+L4GyrvRBxIN2QlHpchewxSlK+ok0NNzvQuD7qKDXMUbj/aPMBXerGwdur1obNiAqPsCgoO1y7rxxHXdXJ3BL08W8RBTb2XlfyempBWMWVA","Leaf":{"Level":1,"Index":"HzdAcxzkVDEdCnrc0Rq6KZ7zewuqgy6BZrBBXkbPiL4=","Value":"a2V5ZWtlcg==","IsEmpty":false,"Commitment":{"Salt":"NIKu0xbTVEsBREihmzxiLqWKmv6m0pu5mapCuHw2YBA=","Value":"b0uh+7ssMwvp7xcN+lm97tWsRABJr1cjS+dAUNty5Us="}}},"STR":{"TreeHash":"QgJG9j1yq3z50PhddRKkA34V+8ucEqWWjzDnyv6y960=","Epoch":3,"PreviousEpoch":2,"PreviousSTRHash":"6txKUQyf+TCLYkrwS0qEaN4LI6Pn2rV7oavWgUaXEeY=","Signature":"PfJvZuT4MPSBj1lJ2iU6qlD81r6VMh4KtH83a5HqGsQd9b4bi7CFKTzDjlxy0TgK+HE+lv+4shZ222NgZJdiCg==","Policies":{"Version":"0.3","HashID":"SHAKE128","VrfPublicKey":"Csmr5OOqALXvRq/rND+A6pYaEQ0tA7x0sSMPML+N8vQ=","EpochDeadline":60}}}}`
```

Comment:
This string is likely to go stale as we update the code.  It would be preferable to generate the response.